src/utils/index.js

```diff
@@ -1,14 +1,29 @@
 var d3 = require('d3');
 
 
-exports.calculateScales = (chartWidth, chartHeight, xValues, yValues) => {
+Array.prototype.unique = function() {
+  var o = {}, i, l = this.length, r = [];
+  for(i=0; i<l;i+=1) o[this[i]] = this[i];
+  for(i in o) r.push(o[i]);
+  return r;
+};
```

Comment:
It's generally not good to extend native prototypes with non-standard methods in library code.